Adding fetch for package bundle size, using bundlephobia

JonLim opened the pull request:
* Added a new file, `fetchBundlesSizes.js`, that sends a request to the Bundlephobia api endpoint for fetching sizes of packages
* Added conditional section to package names displayed in header, to add bundle size if it exists for package after the package name

![image](https://user-images.githubusercontent.com/82186/95495697-c2402100-096d-11eb-8cc7-395247e9d5e7.png)
(This is with the size data wrapped with `<small>` tags, open for pulling that back!)

cheapsteak commented on packages/frontend/src/home/home.html:
```diff
@@ -42,7 +42,12 @@ <h1 class="heading">
-          >{{ moduleName }}</a>
+          >
+          {{ moduleName }}
+          <template v-if="packagesBundleSizesResponse && packagesBundleSizesResponse[moduleName]">
+            <small>({{ packagesBundleSizesResponse[moduleName]["gzip"] }} gzipped)</small>
+          </template>
```
(not important) I think the `<template>` can be a `<small>` here (I think you can add `v-if`s anywhere)

JonLim replied:
Ooh, good call, easy adjustment.

cheapsteak reviewed (approved):
yay! been missing this, thank you!

some small style tweaks I'd love to follow up with but doesn't need to block this getting out!

i'm thinking of adding this to the `<small>` tags
```css
    font-size: 0.7em;
    font-weight: 500;
    margin-left: -0.2em;
```
and dropping the trailing "download trends" 
![image](https://user-images.githubusercontent.com/743976/95498283-92931800-0971-11eb-9397-3db3228ee7da.png)
![image](https://user-images.githubusercontent.com/743976/95498196-742d1c80-0971-11eb-99fa-4906881f4fd0.png)

JonLim commented:
> yay! been missing this, thank you!
> 
> some small style tweaks I'd love to follow up with but doesn't need to block this getting out!
> 
> i'm thinking of adding this to the `<small>` tags
> 
> ```css
>     font-size: 0.7em;
>     font-weight: 500;
>     margin-left: -0.2em;
> ```
> 
> and dropping the trailing "download trends"
> ![image](https://user-images.githubusercontent.com/743976/95498283-92931800-0971-11eb-9397-3db3228ee7da.png)
> ![image](https://user-images.githubusercontent.com/743976/95498196-742d1c80-0971-11eb-99fa-4906881f4fd0.png)

I can make those changes, if you like!

Other alternative to removing "download trends" is maybe moving it after "compare"?

```
compare download trends for: <package>
```

But either one works for me, let me know what you think!

cheapsteak commented:
let's get rid of it, less words better :D 

also planning to add more things beyond download trends (already have some now with release stats and bundle sizes!)

JonLim commented:
Changes deployed!

* Removed "download trends"
* Added styling to `<small>`
* Added `v-if` to `<small>` and removed `<template>`

![image](https://user-images.githubusercontent.com/82186/95503608-8f9c2580-0979-11eb-9962-4f0f33ad972e.png)

Looks pretty good!

cheapsteak reviewed (approved):
🙏 🎉 thank you!!
